consensus/spos/bls/export_test.go

```diff
@@ -356,3 +356,13 @@ func (sr *subroundEndRound) GetSentSignatureTracker() spos.SentSignaturesTracker
 func GetStringValue(messageType consensus.MessageType) string {
 	return getStringValue(messageType)
 }
+
+// ChangeEpoch calls the unexported changeEpoch function
+func (sr *subroundStartRound) ChangeEpoch(epoch uint32) {
```

Comment:
not really needed, can be used EpochStartAction with a header with the desired epoch on it